### ISSUE TITLE
This emoji causes subsequent text to be hidden in SVG output

### ISSUE THREAD

**Issue by paulfurley** [nbedos/termtosvg#89]
```
termtosvg
echo "before 🕵️‍ after"
```
Gives this SVG output, where the word "after" is not visible:

https://gist.github.com/paulfurley/4e74ecc062837edcf9a3270849915182

Here's a static screenshot

![image](https://user-images.githubusercontent.com/254290/50839655-ac489b00-1358-11e9-93f2-e5a62b728c28.png)

I'm using termtosvg==0.7.0 on Ubuntu 18.04

BTW thanks for making `termtosvg`, we *love*it! It's extremely useful for illustrating release notes for [Fluidkeys, our command-line app](https://www.fluidkeys.com/blog)

**Comment by paulfurley** [comment]
Can others reproduce this?

**Comment by jedahan** [comment]
Could not reproduce with termtosvg 0.7.0 on macOS 10.14.2 with alacritty 0.2.0, zsh 5.6.2

**Comment by nbedos** [comment]
> Can others reproduce this?

Yes I can reproduce this behavior, I just haven't had time to investigate further. I'll try to look at this in the next few days.

**Comment by Rondom** [comment]
Yes, I can. Using the following script I isolated the issue to two unicode-codepoints in your string that make the issue appear. See also: https://www.fontspace.com/unicode/analyzer/?q=before+%F0%9F%95%B5%EF%B8%8F%E2%80%8D+after

    #!/bin/sh -eu
    u_sleuth="\xf0\x9f\x95\xb5"
    u_emoj_vs16="\xef\xb8\x8f"
    u_zero_width_joiner="\xe2\x80\x8d"

    echo -e "         original test: ABCDEFG${u_sleuth}${u_emoj_vs16}${u_zero_width_joiner}ABCDEFG"
    echo -e "           just sleuth: ABCDEFG${u_sleuth}ABCDEFG"
    echo -e "       just emoj vs-16: ABCDEFG${u_emoj_vs16}ABCDEFG"
    echo -e "just zero-width-joiner: ABCDEFG${u_zero_width_joiner}ABCDEFG"

All lines but the second line (just the sleuth) show the issue for me. Maybe this is a pyte-issue? Do they have some example-app to check?

**Comment by nbedos** [comment]
@Rondom Thanks for investigating. I did some tests and it turns out Pyte drops all data following an "unprintable" character (the zero width joiner of your test case) : https://github.com/selectel/pyte/blob/0.8.0/pyte/screens.py#L522

A simple workaround is to feed Pyte a single character at a time so that if a character is not printable it does not affect the rest of the line. I've implemented this via 1a45282. Let met know if that fixes the issue.
